app/MindWork AI Studio/Components/ChatComponent.razor.cs

```diff
@@ -59,6 +60,7 @@ public partial class ChatComponent : MSGComponentBase, IAsyncDisposable
     private string currentWorkspaceName = string.Empty;
     private Guid currentWorkspaceId = Guid.Empty;
     private CancellationTokenSource? cancellationTokenSource;
+    private bool disableProfile = false; // TODO
```

Comment:
`TODO` comment?

Reply:
Also: Not required, right? The selection of profiles is controlled by `Disabled`, correct?